app/api/v1/mediator/ballot.py

```diff
@@ -33,6 +44,38 @@ def cast_ballot(request: AcceptBallotRequest = Body(...)) -> Any:
     return casted_ballot.to_json_object()
 
 
+@router.post("/decrypt", tags=[TALLY])
+def decrypt_ballots(request: DecryptBallotsRequest = Body(...)) -> Any:
+    ballots = [
+        CiphertextAcceptedBallot.from_json_object(ballot)
+        for ballot in request.encrypted_ballots
+    ]
+    context: CiphertextElectionContext = CiphertextElectionContext.from_json_object(
+        request.context
+    )
+
+    # Build a lookup for each ballot, containing the shares for decryption
+    all_shares: List[BallotDecryptionShare] = [
```

Comment:
feels like this could be helpful function and then wouldn't need the comment or it could use docstring to describe what's happening.